src/config.ts

```diff
@@ -2,6 +2,9 @@ export interface Config {
   host: string;
   timeout?: number;
   strictGDPR?: boolean;
+  headers?: {
+    [key: string]: number | string | boolean;
```

Comment:
I am looking at the headers property in axios types, should we add something similar?

https://github.com/axios/axios/blob/master/index.d.ts

Reply:
Well we could. In general my proposition is to add custom headers option.